src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbyplaintextcommentfilter/InputSuppressWithNearbyPlainTextCommentFilterCheckFormat.bash

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+: '
+SuppressWithNearbyPlainTextCommentFilter
+commentFormat = (default)SUPPRESS CHECKSTYLE (\\w+)
+checkFormat = .*Line.*
+messageFormat = (default)(null)
+idFormat = (default)(null)
+influenceFormat = (default)0
+
+com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck
+fileExtensions = (default)all files
+ignorePattern = (default)^(package|import) .*
+max = (default)80
+
+com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck
+format = this should not appear
+message = (default)(null)
+ignoreCase = (default)false
+minimum = (default)0
+maximum = (default)0
+ignoreComments = (default)false
+
+
+ '
+
+set -e
+
+CHMOD=$(find -type f -not -path '*/\.git/*' \
+          -a -type f -not -name '*.sh' \
+          -a -type f -not -name '*.pl' \
+          -a \( -type d -not -perm 775 -o -type f -executable \))
+if [[ -n $CHMOD ]]; then
```

Comment:
Please remove command above.
We do not need reasonable code here. We need short and concise input for testing

Reply:
Done